Kitodo-Docket/src/test/java/org/kitodo/docket/ExportDocketTest.java

```diff
@@ -39,9 +39,14 @@ public void testStartExport() throws IOException {
         FileOutputStream fileOutputStream = new FileOutputStream(file);
         File xslFile = new File("src/test/resources/docket.xsl");
 
-        exportDocket.startExport(docketDataGenerator.createDocketData("processId", "signature", "doctype"),
-                fileOutputStream, xslFile);
-        fileOutputStream.close();
+        try {
```

Comment:
Please use [try-with-resources](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) statement instead of handling this manually.